python/cugraph-pyg/cugraph_pyg/data/cugraph_store.py

```diff
@@ -367,6 +368,15 @@ def __infer_offsets(
             }
         )
 
+    def __dask_array_from_numpy(
+        self, array: np.ndarray, client: distributed.client.Client, npartitions: int
```

Comment:
Do we need `client` argument here as it's not used explicitly?

Reply:
no, I forgot to take that out after updating the function.

Reply:
should be fixed now